src/System.Drawing.Common/src/System/Drawing/Pen.cs

```diff
@@ -61,9 +57,7 @@ public Pen(Color color, float width)
                                                 width,
```

Comment:
This isn't really necessary, but I hate how these blocks are indented (in the existing code). If you are fixing up formatting in the code, feel free to change it like this:

```C#
int status = SafeNativeMethods.Gdip.GdipCreatePen1(
    color.ToArgb(), 
    width, 
    (int)GraphicsUnit.World, 
    out pen);
```

I'm just pointing this out in case you notice these in any future PR's. You don't need to update this one.